### PR TITLE
M3-2247: Unit Tests for Clone Configs/Disks

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

More unit tests for the Clone Configs/Disks feature.

You'll probably need to clear out `node_modules` since this doesn't have the MUI update. 

This is the last PR into the feature branch. Once it's merged I'll rebase against `develop`.

**To test:** `yarn test src/features/linodes/CloneLanding/`

### CODE REVIEW

File: src/features/linodes/CloneLanding/Configs.test.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+describe('Configs', () => {
+  it('renders a row for each config', () => {
+    const { getByText } = render(wrapWithTheme(<Configs {...props} />));
+    linodeConfigs.forEach(eachConfig => {
+      expect(getByText(eachConfig.label)).toBeDefined();
+    });
+  });
+
+  it('fires the handle event when clicked', () => {
+    const { getByTestId } = render(wrapWithTheme(<Configs {...props} />));
```
Comment (martinmckenna) [inline src/features/linodes/CloneLanding/Configs.test.tsx]: we should try to use the [rerender](https://testing-library.com/docs/marko-testing-library/api#rerender) logic here and probably only render this once for performance's sake. Tbh, I haven't run any benchmarking tests so I'm not 100% there is a performance gain.

File: src/features/linodes/CloneLanding/Disks.test.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  it('renders an empty state when no configs', () => {
+    const { getByText, getByTestId } = render(
+      wrapWithTheme(<Disks {...props} disks={[]} />)
+    );
+    expect(getByTestId('table-row-empty')).toBeDefined();
+    expect(getByText('No items to display.')).toBeDefined();
+  });
+
+  it('checks the disk if the associated config is selected', () => {
+    const { getByTestId } = render(
```
Comment (martinmckenna) [inline src/features/linodes/CloneLanding/Disks.test.tsx]: same comment. We should render this once any rerender.

File: src/features/linodes/CloneLanding/Configs.test.tsx (hunk 1 above)
Comment (johnwcallahan) [reply src/features/linodes/CloneLanding/Configs.test.tsx]: The trouble with rendering a component once per suite of tests is that the individual tests become tightly coupled. The docs recommend cleaning up the DOM in between each test: 

> The cleanup function should be called between tests to remove the created DOM nodes and keep the tests isolated.

https://testing-library.com/docs/react-testing-library/api#cleanup

AFAICT, this means we need to render a new component for each test.

`rerender` is a nice API for rendering the same component with different props within the context of a single test. 


**TLDR:** the only way to get one render per suite is to remove `afterEach(cleanup)`, which is against the recommendation of the docs.

Comment (martinmckenna) [reply src/features/linodes/CloneLanding/Configs.test.tsx]: fair enough. ✅
